src/middleware/authentication.middleware.ts

```diff
@@ -12,7 +12,11 @@ export const authentication = (req: Request, res: Response, next: NextFunction):
   try {
     if (!checkUserSignedIn(req.session)) {
       logger.infoRequest(req, 'User not authenticated, redirecting to sign in page, status_code=302');
-      return res.redirect(`/signin?return_to=${SOLD_LAND_FILTER_URL}`);
+      if (req.path.startsWith(LANDING_URL)) {
```

Comment:
Should we test for UPDATE_LANDING_URL and turn this if/then arround so that the default is registration?